test/functional/apk-utils-e2e-specs.js

```diff
@@ -0,0 +1,65 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import 'mochawait';
+import ADB from '../../lib/adb.js';
+import path from 'path';
+
+chai.use(chaiAsPromised);
+
+describe('apk utils', () => {
+  let adb = new ADB();
+  const contactManagerPath = path.resolve(__dirname, '..', '..', '..', 'test',
+                                          'ContactManager.apk');
+  const deviceTempPath = '/data/local/tmp/';
+  before(async () => {
+    await adb.createADB();
+  });
+  it('should be able to install/remove app and detect its status', async () => {
+    (await adb.isAppInstalled('foo')).should.be.false;
+    await adb.install(contactManagerPath);
+    (await adb.isAppInstalled('com.example.android.contactmanager')).should.be.true;
+    (await adb.uninstallApk('com.example.android.contactmanager')).should.be.true;
+    (await adb.isAppInstalled('com.example.android.contactmanager')).should.be.false;
+    (await adb.uninstallApk('com.example.android.contactmanager')).should.be.false;
+    await adb.rimraf(deviceTempPath + 'ContactManager.apk');
+    await adb.push(contactManagerPath, deviceTempPath);
+    await adb.installFromDevicePath(deviceTempPath + 'ContactManager.apk');
+  });
+  describe('startApp', async () => {
+    it('should be able to start', async () => {
+      await adb.install(contactManagerPath);
+      await adb.startApp({pkg: 'com.example.android.contactmanager',
+                          activity: 'ContactManager'});
+    });
+    it('should throw error for wrong activity', async () => {
+      await adb.install(contactManagerPath);
+      await adb.startApp({pkg: 'com.example.android.contactmanager',
+                          activity: 'ContactManage'}).should.eventually
+                                                     .be.rejectedWith('Activity');
+    });
+    it('should throw error for wrong wait activity', async () => {
+      await adb.install(contactManagerPath);
+      await adb.startApp({pkg: 'com.example.android.contactmanager',
+                          activity: 'ContactManager',
+                          waitActivity: 'ContactManage',
+                          waitDuration: 1000}).should.eventually
+                          .be.rejectedWith('ContactManage');
+    });
+    it('should start activity with wait activity', async () => {
+      await adb.install(contactManagerPath);
+      await adb.startApp({pkg: 'com.example.android.contactmanager',
+                          activity: 'ContactManager',
+                          waitActivity: '.ContactManager',
+                          waitDuration: 1000});
```

Comment:
Is there anything that can be asserted here? Or will the `adb.startApp` call fail the test if it breaks?